luxonis_ml/nn_archive/config.py

```diff
@@ -6,7 +6,7 @@
 
 from .model import Model
 
-CONFIG_VERSION = Literal["1.0"]
+CONFIG_VERSION = Literal["1.1"]
```

Comment:
This should be changed it to `Literal["1.0", "1.1"]`. Otherwise, we won't be able to load old NN Archive configs after this is merged.